lte/gateway/fabfile.py

```diff
@@ -500,13 +498,8 @@ def _oai_coverage():
 def _run_unit_tests():
     """ Run the magma unit tests """
     with cd(AGW_ROOT):
-        # Run the unit tests
-        run('make test')
-
-
-def _python_coverage():
-    with cd(AGW_PYTHON_ROOT):
-        run('make coverage')
+        # Run all unit tests that are not run as pre-commit checks in CI
```

Comment:
would it make sense to change the name of the function and description to make sure future selves know that function is not running all unit test but just few
`run_sudo_requiered_unit_test` (?)

Reply:
good point! will modify.